### ISSUE TITLE
DRG rotation seems to be incorrect.

### ISSUE THREAD

**Issue by si811** [MKhayle/XIVComboPlugin#322]
after spiral blow or lance barrage the combo restarts instead of progressing to Chaotic Spring or Heavens' Thrust.

**Comment by andreszubi** [comment]
Same issue here, Spiral Blow and Lance Barrage are not turning into those abilities it just goes back to True Thrust

**Comment by eclipsaa** [comment]
dragoon buttons dont work anymore.

**Comment by si811** [comment]
if it helps anyone else, I went back to the previous dev build and that one is fine.

**Comment by RyudoGaming** [comment]
can you link me to that build?

**Comment by si811** [comment]
https://github.com/Skooz/XIVComboPlugin/releases

**Comment by RyudoGaming** [comment]
That was quick! Thanks!

**Comment by Ansler** [comment]
Yeah, just got Dragoon to 96. Vorpal Thrust and Disembowel get upgraded to Lance Barrage and Spiral Blow and XiV Combo's rotations stop working properly, just restarting to true thrust after using the new moves.
